app.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+
+console.log('-----First and Pike Begins----');
+var firstAndPike = {
+  name: 'First and Pike',
+  minCustomer: 23,
+  maxCustomer: 65,
+  avgCookies: 6.3,
+  avgHours: [],
+  cookiesPerHour: function() {
+    for (var hours = 0; hours < 15; hours++) {
+      console.log('Inside loop ' + hours);
+      var mathStorage = Math.floor(Math.random() * (this.maxCustomer + 1 - this.minCustomer) + this.minCustomer) * this.avgCookies;
+      this.avgHours.push(mathStorage);
+    }
+  }
+};
+
+console.log(firstAndPike.cookiesPerHour());
+
+var sectEl = document.getElementById('cookie-list');
+
+var listElement = document.createElement('ul');
```

Comment:
This is super picky, but it may make sense to wrap the ul and the h1 inside of a div or article or something so that everything for a particular store is wrapped in a single, root element unique to that store, then that root element gets put into the cookie-list section. Hopefully that makes sense.